semcore/badge/src/Badge.jsx

```diff
@@ -15,10 +16,16 @@ class RootBadge extends Component {
 
   render() {
     const SBadge = Root;
-    const { styles, color, bg } = this.asProps;
+    const { styles, color, bg, children } = this.asProps;
 
     return sstyled(styles)(
-      <SBadge render={Box} tag="span" use:color={resolveColor(color)} use:bg={resolveColor(bg)} />,
+      <SBadge
+        render={Box}
+        tag="span"
+        aria-label={`badge: ${reactToText(children)}`}
```

Comment:
What `badge` means for components end user?

Reply:
it seems to me that for the voice acting it will be better to sound "position tracking icon new" than "position tracking new". But maybe I'm wrong. We need the opinion of UX specialist